cmd/convox/install.go

```diff
@@ -794,3 +808,44 @@ func readCredentialsFromSTDIN() (creds *AwsCredentials, err error) {
 
 	return &input.Credentials, err
 }
+
+func awsCLICredentials() (*AwsCredentials, error) {
+	data, err := awsCLI("iam", "get-user")
+
+	if err != nil && strings.Index(err.Error(), "executable file not found") > -1 {
+		fmt.Println("Installing the AWS CLI will allow you to install a Rack without specifying credentials.")
+		fmt.Println("See: http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-set-up.html")
+		fmt.Println()
+		return nil, nil
+	}
+
+	if strings.Index(string(data), "Unable to locate credentials") > -1 {
+		fmt.Println("You appear to have the AWS CLI installed but have not configured credentials.")
+		fmt.Println("You can configure credentials by running `aws configure`.")
+		fmt.Println()
+		return nil, nil
+	}
+
+	access, err := awsCLI("configure", "get", "aws_access_key_id")
+
+	if err != nil {
+		return nil, err
+	}
+
+	secret, err := awsCLI("configure", "get", "aws_secret_access_key")
+
+	if err != nil {
+		return nil, err
+	}
+
+	creds := &AwsCredentials{
+		Access: strings.TrimSpace(string(access)),
+		Secret: strings.TrimSpace(string(secret)),
+	}
+
+	return creds, nil
+}
+
+func awsCLI(args ...string) ([]byte, error) {
+	return exec.Command("aws", args...).CombinedOutput()
```

Comment:
Do we need to explicitly pass AWS_DEFAULT_PROFILE into this helper to make sure we use the profile intended by the user and their active terminal session?